test-clients/src/main/java/com/hedera/services/bdd/suites/perf/MixedOpsLoadTest.java

```diff
@@ -0,0 +1,287 @@
+						logIt(ignore -> settings.toString()),
+						newKeyNamed("submitKey"),
+						tokenOpsEnablement(),
+						scheduleOpsEnablement(),
+						cryptoCreate("treasury")
+								.hasRetryPrecheckFrom(permissiblePrechecks)
+								.key(GENESIS)
+				).when(
+						fileUpdate(APP_PROPERTIES)
+								.payingWith(GENESIS)
+								.overridingProps(Map.of("hapi.throttling.buckets.fastOpBucket.capacity", "1300000.0",
+										"hapi.throttling.ops.consensusUpdateTopic.capacityRequired", "1.0",
+										"hapi.throttling.ops.consensusGetTopicInfo.capacityRequired", "1.0",
+										"hapi.throttling.ops.consensusSubmitMessage.capacityRequired", "1.0",
+										"tokens.maxPerAccount", "10000000")),
+						cryptoCreate(sender)
+								.balance(initialBalance.getAsLong())
+								.withRecharging()
+								.key(GENESIS)
+								.rechargeWindow(3)
+								.hasRetryPrecheckFrom(permissiblePrechecks),
+						cryptoCreate(receiver)
+								.hasRetryPrecheckFrom(permissiblePrechecks)
+								.key(GENESIS),
+						createTopic(topic)
+								.submitKeyName("submitKey"),
+						inParallel(IntStream.range(0, NUM_SUBMISSIONS)
+								.mapToObj(ignore ->
+										tokenCreate("token" + tokenId.getAndIncrement())
+												.payingWith(GENESIS)
+												.signedBy(GENESIS)
+												.fee(ONE_HUNDRED_HBARS)
+												.initialSupply(ONE_HUNDRED_HBARS)
+												.treasury("treasury")
+												.hasRetryPrecheckFrom(permissiblePrechecks)
+												.hasPrecheckFrom(DUPLICATE_TRANSACTION, OK)
+												.deferStatusResolution()
+												.noLogging())
+								.toArray(n -> new HapiSpecOperation[n])),
+						sleepFor(10000),
+						inParallel(IntStream.range(0, NUM_SUBMISSIONS)
+								.mapToObj(ignore ->
+										scheduleCreate("schedule" + scheduleId.getAndIncrement(),
+												cryptoTransfer(tinyBarsFromTo(sender, receiver, 1))
+														.signedBy(sender)
+														.payingWith(DEFAULT_PAYER)
+														.hasPrecheckFrom(STANDARD_PERMISSIBLE_PRECHECKS)
+														.deferStatusResolution()
+										)
+												.signedBy(DEFAULT_PAYER)
+												.fee(ONE_HUNDRED_HBARS)
+												.inheritingScheduledSigs()
+												.hasPrecheckFrom(STANDARD_PERMISSIBLE_PRECHECKS)
+												.deferStatusResolution()
+												.adminKey(DEFAULT_PAYER)
+												.noLogging())
+								.toArray(n -> new HapiSpecOperation[n])),
+						sleepFor(10000),
+						inParallel(IntStream.range(0, NUM_SUBMISSIONS)
+								.mapToObj(i -> tokenAssociate(sender, "token" + i)
+										.payingWith(GENESIS)
+										.signedBy(GENESIS)
+										.hasRetryPrecheckFrom(permissiblePrechecks)
+										.hasPrecheckFrom(DUPLICATE_TRANSACTION, OK)
+										.hasKnownStatusFrom(SUCCESS,
+												TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT,
+												INVALID_TOKEN_ID,
+												TRANSACTION_EXPIRED,
+												TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED,
+												FAIL_INVALID,
```

Comment:
`FAIL_INVALID` is specifically reserved for something severe as a response from a server, so a client who receives this should fail hard

Reply:
@qnswirlds  Thank you ! Removed it